server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreRestoreIT.java

```diff
@@ -309,7 +316,7 @@ public void testRTSRestoreWithCommittedDataMultipleIndicesPatterns() throws IOEx
      * @throws IOException IO Exception.
      */
     @AwaitsFix(bugUrl = "https://github.com/opensearch-project/OpenSearch/issues/8480")
-    public void testRTSRestoreWithCommittedDataDefaultAllIndices() throws IOException {
+    public void testRTSRestoreWithCommittedDataDefaultAllIndices() throws Exception {
```

Comment:
Unable to pull line 345 below in this comment, but given that this IT validates that by default if no indices are passed for restore the API should consider all remote indices for restore, should we not pass `indices` to `restore()`?

Reply:
We can make this change when we are fixing https://github.com/opensearch-project/OpenSearch/issues/8480